music/cmd/sidecar_cmds.go

```
@@ -0,0 +1,279 @@
+					addrs, port, extra))
+			}
+		}
+		fmt.Printf("%s\n", columnize.SimpleFormat(out))
+	},
+}
+
+var sidecarHelloCmd = &cobra.Command{
+	Use:   "hello",
+	Short: "Send a hello message to all sidecars for a zone",
+	Run: func(cmd *cobra.Command, args []string) {
+		PrepArgs("zonename")
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)